iib/web/models.py

```diff
@@ -106,6 +106,29 @@ class BundleDeprecation(db.Model):
     )
 
 
+class AddRequestBundleDeprecation(db.Model):
```

Comment:
Would be worth it to merge it with th class BundleDeprecation(db.Model)? or create a general one for deprecation?

Reply:
So the BundleDeprecation is an association table between the merge-index-image requests and bundle images. This one is an association between the Add requests and bundle images. I think we can rename BundleDeprecation to MergeIndexImageRequestBundleDeprecation to reduce confusion as a separate commit.

Reply:
it would make less confusion if you rename it. thank you